cfme/configure/access_control/__init__.py

```diff
@@ -452,21 +471,18 @@ class GroupForm(ConfigurationView):
     cancel_button = Button('Cancel')
     retrieve_button = Button('Retrieve')
 
-    @View.nested
-    class my_company_tags(Tab):  # noqa
-        """ Represents 'My company tags' tab in Group Form """
-        TAB_NAME = "My Company Tags"
-        tree_locator = 'tags_treebox'
-        tree = CbTree(tree_locator)
+    my_company_tags = View.nested(MyCompanyTags)
+    hosts_and_clusters = View.nested(Hosts_And_Clusters)
+    vms_and_templates = View.nested(Vms_And_Templates)
 
     @View.nested
-    class hosts_and_clusters(Tab):  # noqa
+    class Hosts_And_Clusters(Tab):  # noqa
```

Comment:
@rlbabyuk This looks like these tab classes are duplicates, now that you've defined them above and used `View.nested(<viewclass>)` on L474, 475, 476. Are they serving some other purpose or just didn't get removed?